safe_transaction_service/history/signals.py

```diff
@@ -247,9 +247,13 @@ def process_webhook(
     )
     for payload in payloads:
         if address := payload.get("address"):
-            send_webhook_task.delay(address, payload)
+            send_webhook_task.apply_async(
+                args=(address, payload), priority=1
+            )  # Almost the lowest priority
             if is_relevant_notification(sender, instance, created):
-                send_notification_task.apply_async(args=(address, payload), countdown=5)
+                send_notification_task.apply_async(
+                    args=(address, payload), countdown=5, priority=1
```

Comment:
Wouldn't we need to configure `app.conf.broker_transport_options` to make Redis aware of the priority queues?

> To start scheduling tasks based on priorities you need to configure queue_order_strategy transport option.

https://docs.celeryproject.org/en/stable/userguide/routing.html#redis-message-priorities

Reply:
We already do, take a look at `base.py` line 234